config/locales/en.yml

```diff
@@ -35,6 +35,7 @@ en:
       previous: Previous
       first: First
       truncate: '&hellip;'
+    session: Log on
```

Comment:
Il faut qu'on décide de la terminologie :

- Log on / Log off / Register
- Log in / Log out / Register
- Sign in / Sign out / Register
- Sign in / Sign out / Sign up

Vous voyez d'autres alternatives ?

J'ai un penchant pour Sign in / Sign out / Sign up mais ça se discute. En cherchant rapidement je n’ai pas trop trouvé de consensus.

Reply:
https://blog.benjamincharity.com/log-in-vs-login-vs-sign-in/

L'étude de Lee Munroe date, mais est effectivement exhaustive et intéressante :)

Reply:
Bon il y a un bon argument qui revient souvent (et qui contredis ma préférence) c'est de na pas utiliser "sign in"/ "sign up" car trop proches et peuvent facilement induire en erreur. J’ajoute une autre option à la liste.

Reply:
Pour moi, ça me semble plus juste d'utiliser (selon mon Robert & Collins) : 
Log on / Log off / Register

Néanmoins, il est plus courant de voir Sign in / Sign out / Create an Account

Reply:
Moi j'aime bien "Sign in / Sign out / Create an account" ; j'ai l'impression que ca utilise un vocabulaire courant, tout en réduisant l'ambiguité du "Sign-up".